dwitter/templates/dweet/dweet.html

```diff
@@ -39,6 +47,38 @@
           case "hideStats":
             setStatsVisibility(false);
             break;
+          case "record":
+            recording = true;
```

Comment:
Check if recording is already true here? It might be possible to send two "record" messages in a row if you click fast?

Reply:
It can only send a stop message after you press record. The message depends on wheter the record button has the class name "recording" and thag's the first thing I change

Reply:
Fair enough. In this case because you do the toggle of the class directly in the click event I don't see way you will get anything but every other 'record' and 'stop', as js is single-threaded in this case.

I was assuming there was a bounce to the actual dweet confirming that it is indeed recording before switching. I guess this means it will tell you it is recording even if it went wrong :P I think it's fine as it is then.